airflow/api_connexion/schemas/task_schema.py

```diff
@@ -58,6 +58,7 @@ class TaskSchema(Schema):
     sub_dag = fields.Nested(DAGSchema, dump_only=True)
     downstream_task_ids = fields.List(fields.String(), dump_only=True)
     params = fields.Method('get_params', dump_only=True)
+    is_mapped = fields.Boolean(dump_only=True)
```

Comment:
Adding a field to the schema also requires adding it to the API definition in `airflow/api_connexion/openapi/v1.yaml` (line 2,938 onwards), so that it shows up in the docs.

Reply:
Thanks, I though it was automatically generated. Added now.